frontend/src/FirstRun/AuthenticationRequiredModalContent.js

```diff
@@ -63,20 +63,22 @@ function AuthenticationRequiredModalContent(props) {
           className={styles.authRequiredAlert}
           kind={kinds.WARNING}
         >
-          {translate('AuthenticationRequiredWarning')}
+          {translate('AuthenticationRequiredWarning', { appName: 'Sonarr' })}
         </Alert>
 
         {
           isPopulated && !error ?
             <div>
               <FormGroup>
-                <FormLabel>{translate('Authentication')}</FormLabel>
+                <FormLabel>{translate('AuthenticationMethod')}</FormLabel>
 
                 <FormInputGroup
                   type={inputTypes.SELECT}
                   name="authenticationMethod"
                   values={authenticationMethodOptions}
-                  helpText={translate('AuthenticationMethodHelpText')}
+                  helpText={translate('AuthenticationMethodHelpText', { appName: 'Sonarr' })}
+                  helpTextWarning={authenticationMethod.value === 'none' ? translate('AuthenticationMethodHelpTextWarning') : undefined}
+                  helpLink="https://wiki.servarr.com/sonarr/faq-v4#forced-authentication"
```

Comment:
this can be 
```suggestion
                  helpLink="https://wiki.servarr.com/sonarr/faq#forced-authentication"
```

v3 faq has an entry at the bottom with the link to the v4 faq; this means once v4 is stable we wont need to update this

Reply:
👍 Seems reasonable to future proof it